Neos.ContentGraph.DoctrineDbalAdapter/src/DoctrineDbalContentGraphProjection.php

```diff
@@ -150,6 +151,11 @@ public function reset(): void
         $this->getState()->forgetInstances();
     }
 
+    public function markStale(): void
+    {
+        $this->getState()->forgetInstances();
+    }
```

Comment:
I don't understand why this can be removed?!

Reply:
not removed. It has been copied from the decorator `ContentGraphProjection` back to the `DoctrineDbalContentGraphProjection`

(basically reverting what we tree did when we watched over christians shoulder in the the content graph thing)

Reply:
oh man.. I read the diff backwards.. somehow.. Time for weekend :)